oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/binary/BinaryAccessTest.java

```diff
@@ -103,6 +107,14 @@ private static URI expectedDownloadURI() {
         return toURI(DOWNLOAD_URL);
     }
 
+    private static String DEFAULT_CDN_DOWNLOAD_URL = null;
+
+    private static String ALT_CDN_DOWNLOAD_URL = "http://cdn.com/dumm/url/for/test/download";
+
+    private static String CDN_DOWNLOAD_URL = DEFAULT_CDN_DOWNLOAD_URL;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5115210)